### PR TITLE
The Butt Update part 2: Fartvengeance

### PR DESCRIPTION
**Pull request by PopNotes**
:cl: PopNotes
add: A bunch more fart messages
soundadd: Two shiny new fartsounds
/:cl:

### CODE REVIEW

File: hippiestation/code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -191,4 +223,6 @@
 				user.gib() //can you belive I forgot to put this here?? yeah you need to see the message BEFORE you gib
 				new /obj/effect/immovablerod/butt(B.loc, locate(endx, endy, 1))
 				priority_announce("What the fuck was that?!", "General Alert")
-				qdel(B)
\ No newline at end of file
+				qdel(B)
+
```
Comment (ghost) [inline hippiestation/code/modules/mob/living/carbon/human/emote.dm]: nice empty line

File: hippiestation/code/modules/mob/living/carbon/human/emote.dm [hunk 2]
```diff
@@ -191,4 +223,6 @@
 				user.gib() //can you belive I forgot to put this here?? yeah you need to see the message BEFORE you gib
 				new /obj/effect/immovablerod/butt(B.loc, locate(endx, endy, 1))
 				priority_announce("What the fuck was that?!", "General Alert")
-				qdel(B)
\ No newline at end of file
+				qdel(B)
+				
+
```
Comment (ghost) [inline hippiestation/code/modules/mob/living/carbon/human/emote.dm]: nice empty line

File: hippiestation/code/modules/butts/main.dm [hunk 3]
```diff
@@ -10,7 +10,7 @@
 	throwforce = 5
 	throw_speed = 4
 	force = 5
-	hitsound = 'hippiestation/sound/effects/fart.ogg'
+	hitsound = pick('hippiestation/sound/effects/fart.ogg','hippiestation/sound/effects/fart2.ogg','hippiestation/sound/effects/fart3.ogg')
```
Comment (tortellinitony) [inline hippiestation/code/modules/butts/main.dm]: You can't put a pick statement in the definition.

**Comment by PopNotes** [comment]
unfucked it for the last time, merge this so I can delete and re-fork

**Comment by PopNotes** [comment]
okay I think I got it all figured out

File: hippiestation/code/modules/butts/main.dm [hunk 4]
```diff
@@ -11,7 +11,6 @@
 	throw_speed = 4
 	force = 5
 	hitsound = 'hippiestation/sound/effects/fart.ogg'
-	body_parts_covered = HEAD
```
Comment (Carbonhell) [inline hippiestation/code/modules/butts/main.dm]: FUN REMOVAL ALERT
PERMABAN THIS GUY

Comment (PopNotes) [inline hippiestation/code/modules/butts/main.dm]: somehow managed to remove that while fucking with the hitsound

File: hippiestation/code/modules/butts/main.dm [hunk 5]
```diff
@@ -87,11 +90,11 @@
 /obj/item/organ/butt/throw_impact(atom/hit_atom)
 	..()
 	var/mob/living/carbon/M = hit_atom
-	playsound(src, 'hippiestation/sound/effects/fart.ogg', 50, 1, 5)
+	playsound(src, pick('hippiestation/sound/effects/fart.ogg','hippiestation/sound/effects/fart2.ogg','hippiestation/sound/effects/fart3.ogg'), 50, 1, 5)
 	if((ishuman(hit_atom)))
 		M.apply_damage(5, STAMINA)
 		if(prob(5))
-			M.Weaken(3)
+			M.Knockdown(60)
```
Comment (Carbonhell) [inline hippiestation/code/modules/butts/main.dm:98]: why this

Comment (PopNotes) [inline hippiestation/code/modules/butts/main.dm]: i don't fucking know, that's how it is in the current hippie file, pretty sure I just have an outdated master

**Pull request closed without merging**